status/backend.json

```
@@ -24,6 +24,9 @@
       "enabled": true,
       "merchants_v2": false
     },
+    "fims": {
+      "enabled": true
```

Is it ready?

---

nope, it was only CP&Paste from another FF
```suggestion
      "enabled": false
```